k8s/elastic-gke-logging/schema.yaml

```diff
@@ -1,4 +1,43 @@
-application_api_version: v1beta1
+x-google-marketplace:
+  schemaVersion: v2
+
+  applicationApiVersion: v1beta1
+
+  publishedVersion: "$TAG"
+  publishedVersionMetadata:
+    releaseNote: >-
+      A regular update.
+    releaseTypes:
+      - Feature
+    recommended: false
+
+  managedUpdates:
+    kalmSupported: false
```

Comment:
A conflict - KALM should be supported (in application.yaml we mention this)

Reply:
This diff was due to wrong base. Outdated